miio/fan.py

```diff
@@ -5,10 +5,60 @@
 import click
 
 from .click_common import command, format_output, EnumType
-from .device import Device
+from .device import Device, DeviceException
 
 _LOGGER = logging.getLogger(__name__)
 
+MODEL_FAN_V2 = 'zimi.fan.v2'
+MODEL_FAN_V3 = 'zimi.fan.v3'
+
+AVAILABLE_PROPERTIES = {
+    MODEL_FAN_V2: [
+        'temp_dec',
+        'humidity',
+        'angle',
+        'speed',
+        'poweroff_time',
+        'power',
+        'ac_power',
+        'battery',
+        'angle_enable',
+        'speed_level',
+        'natural_level',
+        'child_lock',
+        'buzzer',
+        'led_b',
+        'led',
+        'use_time',
+        'bat_charge',
+        'bat_state',
+        'button_pressed',
+    ],
+    MODEL_FAN_V3: [
+        'temp_dec',
+        'humidity',
+        'angle',
+        'speed',
+        'poweroff_time',
+        'power',
+        'ac_power',
+        'battery',
+        'angle_enable',
+        'speed_level',
+        'natural_level',
+        'child_lock',
+        'buzzer',
+        'led_b',
+        'use_time',
+        'bat_charge',
+        'button_pressed',
+    ],
+}
```

Comment:
I think it would be nice to have a `GENERIC_PROPERTIES` (ugh, global variables with uppercasing.. feels so unpythonic :-)), considering quite a few are shared anyway:
```
>>> set(x) & set(y)
{'bat_charge', 'poweroff_time', 'ac_power', 'led_b', 'use_time', 'humidity', 
'button_pressed', 'battery', 'angle_enable', 'speed_level', 'temp_dec',
'child_lock', 'buzzer', 'natural_level', 'power', 'angle', 'speed'}
```